x-pack/docs/en/security/securing-communications/tls-http.asciidoc

```diff
@@ -1,42 +1,59 @@
 
 When {security-features} are enabled, you can optionally use TLS to ensure that
 communication between HTTP clients and the cluster is encrypted.
 
 NOTE: Enabling TLS on the HTTP layer is strongly recommended but is not required.
 If you enable TLS on the HTTP layer in {es}, then you might need to make
-configuration changes in other parts of the Elastic Stack and in any {es}
-clients that you use.
+configuration changes in other parts of the {stack} and in any {es} clients that
+you use.
 
 . If you have not done so already, <<node-certificates,generate node certificates>>.
++
+--
+In particular, you need the files that are generated by the following command:
+
+[source,shell]
+----------------------------------------------------------
+bin/elasticsearch-certutil http
+----------------------------------------------------------
+
+This command generates a zip file that contains certificates and keys for use in
+{es} and {kib}. Each folder contains a readme that explains how to use the files.
+--
+
+. Verify that you've copied the output files to the appropriate locations, as
+specified in the readme files. 
++
+--
+For example, copy the `http.p12` file from the `elasticsearch` folder into a 
+directory within the {es} configuration directory on each node. If you chose to 
+generate one certificate per node, copy the appropriate `http.p12` file to each
+node. If you want to use {kib} to access this cluster, copy the
+`elasticsearch-ca.pem` file from the `kibana` folder into the {kib}
+configuration directory.
+--
 
 . Enable TLS and specify the information required to access the node’s
-certificate.
+certificate. Follow the instructions in the readme in the `elasticsearch` folder
+of the zip file. For example:
 
-** If the certificate is in PKCS#12 format, add the following information to the
-`elasticsearch.yml` file on each node:
+.. Update the `elasticsearch.yml` file on each node with the location of the
+certificates.
 +
 --
+If the certificates are in PKCS#12 format:
+
 [source, yaml]
 --------------------------------------------------
 xpack.security.http.ssl.enabled: true
-xpack.security.http.ssl.keystore.path: certs/elastic-certificates.p12 <1>
-xpack.security.http.ssl.truststore.path: certs/elastic-certificates.p12 <2>
+xpack.security.http.ssl.keystore.path: "http.p12"
 --------------------------------------------------
-<1> If you created a separate certificate for each node, then you might need to
-customize this path on each node. If the filename matches the node name, you can
-use the `certs/${node.name}.p12` format, for example.
-<2> The `elasticsearch-certutil` output includes the CA certificate inside the
-PKCS#12 keystore, therefore the keystore can also be used as the truststore.
-This name should match the `keystore.path` value.
---
 
-** If the certificate is in PEM format, add the following information to the
-`elasticsearch.yml` file on each node:
-+
---
+If the certificates are in PEM format:
```

Comment:
I think this section might be slightly confusing now. We document with the assumption that `bin/elasticsearch-certutil http` is used and suggest accordingly but there is no possibility for the tool to ouput PEM files like `node.key` and `node.crt` that we use in this example. 
`node.crt` and `node.key` also match the examples we use in the transport ssl section above, which might add to the confusion. 


Maybe if we remove "Follow the instructions in the readme in the elasticsearch folder of the zip file. " from step 3, that will make the step more generic. If users have used the `http` subcommand, they have copied the files already ( as suggested in previous step ) , if not, they have their keys and certificates ready at hand. We can still use `http.p12` for consistency here and also use `node1_http.key` and `node1_http.crt` in the PEM example to denote that these are usually different than the ones used for transport TLS. WDYT ?

Reply:
Since the tool supports generating CSRs, and outputs a PEM key for that, we need this section, but I agree the wording is a little awkward.
That said, people know what type of files they have (or at least, if they don't know then I don't think restructuring the content will really help), so I'm OK with it as is.

Something simple like
```suggestion
If you have certificates in PEM format:
```
would be a little clearer, but it's a tiny difference.

Reply:
Ι'm fine with this also now that we changed the key/certificate file names here